content/en/docs/concepts/workloads/pods/pod-overview.md

```diff
@@ -57,49 +57,58 @@ A Pod can specify a set of shared storage {{< glossary_tooltip text="Volumes" te
-* [DaemonSet](/docs/concepts/workloads/controllers/daemonset/)
+* {{< glossary_tooltip text="Deployment" term_id="deployment" >}}
+* {{< glossary_tooltip text="StatefulSet" term_id="statefulset" >}}
+* {{< glossary_tooltip text="DaemonSet" term_id="daemonset" >}}
 
-In general, Controllers use a Pod Template that you provide to create the Pods for which it is responsible.
+Workload resources use a _pod template_, that you provide, to create the Pods for which it is responsible.
 
-## Pod Templates
+## Pod templates
 
-Pod templates are pod specifications which are included in other objects, such as
-[Replication Controllers](/docs/concepts/workloads/controllers/replicationcontroller/), [Jobs](/docs/concepts/jobs/run-to-completion-finite-workloads/), and
-[DaemonSets](/docs/concepts/workloads/controllers/daemonset/).  Controllers use Pod Templates to make actual pods.
-The sample below is a simple manifest for a Pod which contains a container that prints
-a message.
+Pod templates are pod specifications that are are included in other objects, such as
+[Deployments](/docs/concepts/workloads/controllers/deployment/),
+[Jobs](/docs/concepts/jobs/run-to-completion-finite-workloads/), and
+[DaemonSets](/docs/concepts/workloads/controllers/daemonset/).
+Each controller for a workload resource uses the pod template inside the workload object to make actual Pods.
+
+The sample below is a simple manifest for a Job with a pod template that starts one container. The container in that Pod prints a message then pauses.
 
 ```yaml
-apiVersion: v1
-kind: Pod
+apiVersion: batch/v1
+kind: Job
 metadata:
-  name: myapp-pod
-  labels:
-    app: myapp
+  name: hello
 spec:
-  containers:
-  - name: myapp-container
-    image: busybox
-    command: ['sh', '-c', 'echo Hello Kubernetes! && sleep 3600']
+  template:
+    # This is the pod template
+    spec:
+      containers:
+      - name: hello
+        image: busybox
+        command: ['sh', '-c', 'echo "Hello, Kubernetes!" && sleep 3600']
+      restartPolicy: OnFailure
+    # The pod template ends here
 ```
 
-Rather than specifying the current desired state of all replicas, pod templates are like cookie cutters. Once a cookie has been cut, the cookie has no relationship to the cutter. There is no "quantum entanglement". Subsequent changes to the template or even switching to a new template has no direct effect on the pods already created. Similarly, pods created by a replication controller may subsequently be updated directly. This is in deliberate contrast to pods, which do specify the current desired state of all containers belonging to the pod. This approach radically simplifies system semantics and increases the flexibility of the primitive.
+Pod templates work like cookie cutters: once a cookie has been cut, the cookie has no relationship to the cutter.
+If you change the pod template for an existing workload, there is no direct effect on the Pods that are already running.
```

Comment:
- line 70: Capitalize node vs. Node?
- line 88: pod template vs PodTemplate vs Pod template?
- Does it help to bring in `replicas` (copies from the original pattern) when creating the analogy?
`specifying the current desired state of all replicas, pod templates are like cookie cutters`
Line 107: For me, the new text needs some of the original text to make the analogy work. This line seems explicit:
`Modifying the Pod template or even switching to a new template has no direct effect on the running (created?) Pods.`
- nit: s/cutter/cookie cutter/
- It seems as if there should be some glue to introduce the content on line 109,  `Similarly, pods created by a replication controller may subsequently be updated directly.` in contrast to what (how Pods were created before)?
- Is `workload` resource introduced/defined as a concept?
- Line 111: These sentences contain a lot of jargon (`separation of concerns`). What value does the content about nodes and the kubelet add?
Line 109: Capitalize Deployment, capitalize PodTemplate?

Reply:
Think this is resolved.